stylesheets/components/_icon.scss

```diff
@@ -2,7 +2,7 @@
 // This file contains all styles related to icons
 // -----------------------------------------------------------------------------
 
-.material-icons {
+@mixin material-icon {
```

Comment:
Can you explain why it's useful to make this a mixin?

Reply:
Good catch - this was an unfinished thought. I noticed in a lot of places we're using the `.material-icon` class and then overriding font-size and line-height, so my thought was to add arguments to the mixin to make this a little more terse. I've now actually added those arguments...

Reply:
I like it! Would it make sense to apply this approach to component styles that have already been merged to development (I think only pagination?) in this PR, or go back and apply after?

Reply:
I think we can go back and apply afterwards - there might be other arguments we want to pass too. Let's maybe file an issue so we don't forget?